gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java

```diff
@@ -333,7 +337,7 @@ private static DependencyGraph createDependencyGraph(ResolvedConfiguration confi
     return graph;
   }
 
-  private static ClassPathResult createClassPathResult(ResolvedConfiguration configuration) {
```

Comment:
Removing static because it accesses createDependencyGraph, which is now non-static.